src/content/tutorials/fleet-management/cluster-management/migration-to-cluster-api/_index.md

```diff
@@ -56,6 +56,69 @@ Our engineers will check that all resources and infrastructure are correctly mig
+
+In CAPA, since the cluster domain name changes from Vintage, a new Service Account issuer is also introduced. To make for a smooth migration though, we support defining multiple issuers in a cluster. This way, Service Account tokens issued by all the defined issuers will be accepted in the cluster's Kubernetes API. The accepted issuers are defined in `cluster.providerIntegration.controlPlane.kubeadmConfig.clusterConfiguration.apiServer.serviceAccountIssuers` in the cluster App values (order matters as we'll see below).
+
+After a cluster is migrated to CAPI, the Vintage Service Account issuer needs to be phased out, since it's tied to the Vintage cluster domain name, which is also to be phased out eventually. This is a gradual, multi-step process that will require rolling the master nodes in multiple phases.
+
+1. Update the trust policy for all the AWS IAM Roles used by a Service Account via IRSA. The trust policy should allow `sts:AssumeRoleWithWebIdentity` for both Vintage and CAPA issuers during the transition period. This is how the trust policy of the IAM roles would look like (example, make sure to use the correct AWS account id, issuer domains and `ServiceAccount` references):
+
+   ```json
+   {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "Federated": "arn:aws:iam::1234567890:oidc-provider/irsa.foo.k8s.vintage.acme.net"
+                },
+                "Action": "sts:AssumeRoleWithWebIdentity",
+                "Condition": {
+                    "ForAnyValue:StringEquals": {
+                        "irsa.foo.k8s.vintage.acme.net:sub": [
+                            "system:serviceaccount:somenamespace:someserviceaccount"
+                        ]
+                    }
+                }
+            },
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "Federated": "arn:aws:iam::1234567890:oidc-provider/irsa.foo.capi.acme.net"
+                },
+                "Action": "sts:AssumeRoleWithWebIdentity",
+                "Condition": {
+                    "ForAnyValue:StringEquals": {
+                        "irsa.foo.capi.acme.net:sub": [
+                            "system:serviceaccount:somenamespace:someserviceaccount"
+                        ]
+                    }
+                }
+            }
+        ]
+    }
+   ```
+
+2. Switch the order of the Service Account issuers in the cluster values (`cluster.providerIntegration.controlPlane.kubeadmConfig.clusterConfiguration.apiServer.serviceAccountIssuers`). This will instruct the Kubernetes API to start issuing Service Account tokens using the CAPI issuer, while still accepting tokens from the Vintage issuer. Important notes:
```

Comment:
```suggestion
2. Switch the order of the service account issuers in the cluster values (`cluster.providerIntegration.controlPlane.kubeadmConfig.clusterConfiguration.apiServer.serviceAccountIssuers`). This will instruct the Kubernetes API to start issuing service account tokens using the CAPI issuer, while still accepting tokens from the vintage issuer. Important notes:
```